fb_clone/Gemfile

```diff
@@ -40,48 +42,50 @@ gem 'jbuilder', '~> 2.5'
 
 # Reduces boot times through caching; required in config/boot.rb
 gem 'bootsnap', '>= 1.1.0', require: false
-gem 'foundation-rails'
+gem 'bootstrap-will_paginate'
+gem 'carrierwave'
 gem 'faker'
+gem 'foundation-rails'
+gem 'mini_magick'
 gem 'rubocop-faker'
-gem 'carrierwave'           
-gem 'mini_magick'           
-gem 'will_paginate'        
-gem 'bootstrap-will_paginate'
+gem 'will_paginate'
+gem 'font-awesome-rails'
 
 group :production do
-  gem 'pg', '>= 0.18', '< 2.0'
   gem 'fog'
+  gem 'pg', '>= 0.18', '< 2.0'
 end
 group :development, :test do
   # Call 'byebug' anywhere in the code to stop execution and get a debugger console
-  gem 'byebug', platforms: [:mri, :mingw, :x64_mingw]
-  gem "database_cleaner"
-  gem "rspec-rails"
+  gem 'byebug', platforms: %i[mri mingw x64_mingw]
+  gem 'database_cleaner'
   gem 'factory_bot_rails'
   gem 'faker'
+  gem 'rspec-rails'
 end
 
 group :development do
   # Access an interactive console on exception pages or by calling 'console' anywhere in the code.
-  gem 'web-console', '>= 3.3.0'
   gem 'listen', '>= 3.0.5', '< 3.2'
+  gem 'web-console', '>= 3.3.0'
   # Spring speeds up development by keeping your application running in the background. Read more: https://github.com/rails/spring
+  gem 'rails_layout'
   gem 'spring'
   gem 'spring-watcher-listen', '~> 2.0.0'
-  gem 'rails_layout'
 end
 
 group :test do
+  gem 'minitest'
```

Comment:
why `minitest` gem is here?
You have added  `rspec-rails` on your `: development ` group already.

Reply:
I added minitest gem because I have got this warning: https://github.com/ordinaryzelig/capybara_minitest_spec/issues/9